src/react/components/common/tagsInput/tagEditorModal/tagEditorModal.tsx

```diff
@@ -3,21 +3,21 @@ import Form from "react-jsonschema-form";
 import { Button, Modal, ModalBody, ModalHeader } from "reactstrap";
 import { addLocValues } from "../../../../../common/strings";
 import { ITag } from "../../../../../models/applicationState";
-import "./tagEditorModal.scss";
+import ModalFooter from "reactstrap/lib/ModalFooter";
 // tslint:disable-next-line:no-var-requires
 const formSchema = addLocValues(require("./tagEditorModal.json"));
 
 const customStyles = {
```

Comment:
Did you mean to remove this? You mentioned in the description that you removed unnecessary styles from this modal. Looks like this one is still there

Reply:
Weird - i thought i removed it.  Will double check.

Reply:
Fixed import.